src/webpack/patchWebpack.ts

```diff
@@ -171,107 +224,127 @@ function patchFactories(factories: Record<string | number, (module: { exports: a
                         callback(exports.default, id);
                     }
                 } catch (err) {
-                    logger.error("Error while firing callback for webpack chunk", err);
+                    logger.error("Error while firing callback for Webpack subscription", err);
                 }
             }
         } as any as { toString: () => string, original: any, (...args: any[]): void; };
 
-        // for some reason throws some error on which calling .toString() leads to infinite recursion
-        // when you force load all chunks???
-        factory.toString = () => mod.toString();
+        factory.toString = originalMod.toString.bind(originalMod);
         factory.original = originalMod;
 
+        for (const factoryListener of factoryListeners) {
+            try {
+                factoryListener(originalMod);
+            } catch (err) {
+                logger.error("Error in Webpack factory listener", err);
```

Comment:
logging the listener would be useful
```suggestion
                logger.error("Error in Webpack factory listener:\n", err, factoryListener);
```